### PR TITLE
SDI-115 Calendar screen - DESKTOP UI & content changes

### PR DESCRIPTION
**Pull request by steverendell**
accessibility changes

### CODE REVIEW

File: assets/sass/_calendar.scss [hunk 1]
```diff
@@ -200,3 +200,12 @@ ul.calendar {
...
 }
+
+.screenreader-only {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
```
Comment (andymarke) [inline assets/sass/_calendar.scss:211]: I thought GDS has some sort visually hidden class

**Review by andymarke — APPROVED** [review]
LGTM

File: assets/sass/_calendar.scss (hunk 1 above)
Comment (steverendell) [reply assets/sass/_calendar.scss:211]: So it does! ok will push again

**Review by andymarke — APPROVED** [review]
Lgtm

**Pull request merged**